prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -276,12 +282,14 @@ def get_context_user_roles(self):
             )
             logger.info("Context user roles retrieved successfully.")
             return roles
+        except KubernetesSearchAndSaveRolesError as error:
+            logger.critical(str(error))
```

Comment:
Why is this critical here?

Reply:
Because the exception from below is critical too

Reply:
I think we are logging the errors in the above function so there is no need to duplicate the `critical` log.

Reply:
Done